packages/formation-react/src/components/AlertBox/AlertBox.unit.spec.jsx

```diff
@@ -21,13 +21,34 @@ describe('<AlertBox />', () => {
     wrapper.unmount();
   });
 
-  it('should have the expected classname', () => {
+  it('should have the expected classnames', () => {
     const wrapper = shallow(
       <AlertBox content={Content} status="info" isVisible />,
     );
     expect(wrapper.find('.usa-alert').hasClass('usa-alert-info')).to.equal(
       true,
     );
+    expect(
+      wrapper.find('.usa-alert').hasClass('background-color-only'),
+    ).to.equal(false);
+    wrapper.unmount();
+  });
+
+  it('should have have `background-color-only` class added when `backgroundOnly` is `true`', () => {
```

Comment:
Can we add another test to run aXe with the background color prop set? There's a group of axeCheck tests at the end of this file.

Reply:
Done